src/DistributedMetadata/TaskStatusService.cpp

```diff
@@ -136,17 +136,14 @@ Int32 TaskStatusService::append(TaskStatusPtr task)
 {
     auto record = buildRecord(task);
 
-    for (auto i = 0; i < RETRY_TIMES; ++i)
+    const auto & result = dwal->append(record, dwal_append_ctx);
+    if (result.err != ErrorCodes::OK)
     {
-        auto result = dwal->append(record, dwal_append_ctx);
-        if (result.err == ErrorCodes::OK)
-        {
-            return 0;
-        }
+        /// FIXME: Error handling
+        LOG_ERROR(log, "Failed to commit task {}", task->id);
+        return 0;
```

Comment:
`return 0` ?

Reply:
that's the original code, I don't want to modify extra logic without retries in this pr